aleph-client/src/version_upgrade.rs

```diff
@@ -10,6 +10,7 @@ pub fn schedule_upgrade(
     connection: &RootConnection,
     version: Version,
     session: SessionIndex,
+    state: XtStatus,
```

Comment:
technically this requires aleph-client version bump, but I don't know what is the current policy from track-core-chain